.github/renovate.json

```diff
@@ -1,13 +1,21 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["config:base", "schedule:monthly", ":maintainLockFilesWeekly"],
-  "separateMajorMinor": true,
+  "extends": [
+    "config:base",
+    ":combinePatchMinorReleases",
```

Comment:
Any 0. release considers minors to be breaking changes. And since we don't really want to bump patch we should omit those.

Reply:
Should we be using <v1 releases in a production-quality product? How long would that be around for to be worth the customization. Nonetheless this shortcut means `"separateMinorPatch": false` and the opposite could be placed in the cargo-specific package rules lower down.

Reply:
That's how the rust ecosystem works. 0. packages are essentially reasonable stable, they just don't promise api stability. Avoiding all 0. packages means we have to write all the crypto ourselves.